app/assets/javascripts/sitewide/drag_n_drop.js.erb

```diff
@@ -269,20 +269,21 @@
     }
 
     function _uploadedAssetPreview(asset, i) {
-      var html = '<div class="panel panel-default panel-step-attachment-new">';
-      html += '<div class="panel-heading">';
-      html += '<span class="fas fa-paperclip"></span>';
-      html += '<%= I18n.t 'assets.drag_n_drop.file_label' %>';
-      html += '<div class="pull-right">';
-      html += '<a data-item-id="' + i + '" href="#">';
-      html += '<span class="fas fa-times"></span></a></div></div>';
-      html += '<div class="panel-body">';
-      html += '<label class="control-label">';
-      html += '<%= I18n.t 'assets.drag_n_drop.file_label' %>:</label> ';
-      html += truncateLongString(asset.name,
-                                 <%= Constants::FILENAME_TRUNCATION_LENGTH %>);
+      var html = '<div class="attachment-placeholder pull-left new">';
+      html +='<div class="attachment-thumbnail no-shadow %>">';
+      html +='<i class="fas fa-image"></i>';
+      html +='</div>';
+      html +='<div class="attachment-label">' + truncateLongString(asset.name, <%= Constants::FILENAME_TRUNCATION_LENGTH %>);
+      html += '</div>';
+      html +='<div class="spencer-bonnet-modif">';
+      html +='</div>';
+      html +='<div class="remove-icon pull-right">';
+      html +='<a data-item-id="' + i + '" href="#">';
+      html +='<span class="fas fa-trash"></span>';
+      html +='</a>  </div>';
       html += _validateFilesSize(asset);
-      html += '</div></div>';
+      html +='</div>';
```

Comment:
I have a feeling this could be refactored into a partial and included into the page as `hidden`. Readability++, performance? I don't know.

Reply:
Agree. For sure better for readbility, not sure for performance. But I would not change uploading logic at this point, because task is scoped to just update some UI elements..?